### ISSUE TITLE
Utilizing redis WAIT functionality

### ISSUE THREAD

**Issue by Hatched-Kyle** [redis-store/redis-store#344]
Is is possible to utilize the redis WAIT command to help guarantee strong consistency across all nodes?

https://redis.io/commands/wait

**Comment by tubbo** [comment]
I suppose it would be. Were you having an issue with consistency across your replicas?

**Comment by Hatched-Kyle** [comment]
Yeah we're noticing request coming in with expired session data. Even up to 15 seconds after we have updated that specific session. Our redis cluster consists of 4 nodes each with 1 replica.

How are updates pushed to redis? Are they batched together or does every `session[:key] = value` get processed individually?

**Comment by tubbo** [comment]
Well if anything i'd want this to be included in DistributedStore, since that's the class that handles distributed deployments of redis, and i think it handles clusters as well. There aren't any flags you can set in `Redis::Store` for debug logging, you'll just have to add logs in the code if you need them.

**Comment by tubbo** [comment]
@Hatched-Kyle are you potentially having the same issue as https://github.com/redis-store/redis-rack/issues/70?
